src/frontend/src/views/Data/components/TreeActionMenu/components/MoreActionsMenu/index.tsx

```diff
@@ -26,7 +27,8 @@ const MoreActionsMenu = ({
   const isAutoBuild = group?.name === "";
   const isTreeInUserOrg = userGroup?.name === group?.name;
   const canUserDeleteTree = isAutoBuild || isTreeInUserOrg;
-  const isDisabled = status === TREE_STATUS.Started || !canUserDeleteTree;
+  // Fix this to allow users to edit/delete FAILED runs once phylotrees V2 endpoint has been updated to better reflect tree status
```

Comment:
Can you put in the word FIXME in comment just to help it get noticed in the future?

Reply:
oh yep